src/pages/hero/Hero.jsx

```diff
@@ -0,0 +1,204 @@
+                                    <span className="text_line_th">{`$${
+                                      Number(price) + Number(200)
+                                    }`}</span>
+                                  </p>
+                                </div>
+                              </div>
+                            </div>
+                          </Link>
+                        </div>
+                      );
+                    }
+                  )}
+              </div>
+            </div>
+          ))}
+        </div>
+        <div className=" banner_offer">
+          <img src="/assets/banner.png" alt="banner" />
+        </div>
+        <div className="text-center kodchasan hero_title text_cl">
+          <div className="h4">our best products </div>
+          <h2 className="h2">
+            <b>made only for you</b>
+          </h2>
+        </div>
+        {offerData.map((offer) => (
+          <div key={offer} id="head">
+            <div className="flex wrapper_head montserrat">
+              <h2 className="head_text">{offer}</h2>
+              <Link
+                to={`/product/${offer
+                  .replace(/ /g, "")
+                  .replace(/\d+% ?/g, "")
+                  .toLowerCase()}`}
+              >
+                <button className="btn-link view_all p">view all</button>
+              </Link>
+            </div>
+            <div className="wrapper_card flex">
+              {productData
+                .filter((item) => item.offer.toString() === offer)
+                .slice(0, 5)
+                .map(
+                  ({
+                    _id,
+                    name,
+                    image,
+                    price,
+                    productName,
+                    ratings,
+                    inStock,
+                    material,
+                    fastDelivery,
+                    offer,
+                  }) => (
+                    <div key={_id}>
+                      <div className="card_hr flex-center card_hero">
+                        <div className="card_hr_img card_img_hero">
+                          <img src={image} alt={productName} />
+                        </div>
+                        <div className="card_hr_content">
+                          <p title={name} className="h3 card_name_hero">
+                            {name}
+                          </p>
+                          <StarRatings
+                            rating={ratings}
+                            starDimension="2rem"
+                            starSpacing="0px"
+                            starRatedColor="orange"
+                          />
+                          <div className="card_hr_price text_cl">
+                            <p className="p-lg">
+                              deal of the day:
+                              <span className="h2 dod_price">
+                                {` $${parseFloat(price)}`}
```

Comment:
I am not sure why you need 2 $ for template string 😅

Reply:
One is currency '$' and other one is for template string 😅